js/src/main/scala/crystal/react/implicits/package.scala

```diff
@@ -38,17 +40,32 @@ package object implicits {
     def stateIn[F[_]: Sync]: F[S] = self.state.to[F]
   }
 
+  implicit class ModStateSyncIOOps[S](
+    private val self: StateAccess[CallbackTo, S]
+  ) extends AnyVal {
+    // I can't find a generic way to run a `Sync`, which is needed to execute the callback,
```

Comment:
What if you have a `Dispatcher`?

Reply:
A `Dispatcher` can be obtained from an `Async`, or by `IOApp`. CE3 does not provide a `Dispatcher[SyncIO]`. It would probably be overkill.

The new `scalajs-react` will provide a typeclass to specify sync effects with a method to run them. Let's use that when it's available.